### PR TITLE
add validator voting power calculation based on bitmap mask

### PR DESCRIPTION
**Pull request by chaosma**

### CODE REVIEW

File: consensus/quorum/one-node-one-vote.go [hunk 1]
```diff
@@ -43,6 +63,13 @@ func (v *uniformVoteWeight) IsRewardThresholdAchieved() bool {
 	return v.SignersCount(Commit) >= (v.ParticipantsCount() * 9 / 10)
 }
 
+// ComputeTotalPowerByMask computes the total power indicated by bitmap mask
+func (v *uniformVoteWeight) computeTotalPowerByMask(mask *bls_cosi.Mask) *numeric.Dec {
```
Comment (rlan35) [inline consensus/quorum/one-node-one-vote.go]: Use int here. Don't use Dec for uniform vote

File: consensus/quorum/one-node-one-vote.go [hunk 2]
```diff
@@ -33,6 +34,25 @@ func (v *uniformVoteWeight) IsQuorumAchieved(p Phase) bool {
 	return r
 }
 
+// IsQuorumAchivedByMask ..
+func (v *uniformVoteWeight) IsQuorumAchievedByMask(mask *bls_cosi.Mask) bool {
+	threshold := v.QuorumThreshold()
+	currentTotalPower := v.computeTotalPowerByMask(mask)
```
Comment (rlan35) [inline consensus/quorum/one-node-one-vote.go]: Just use: r := v.SignersCount(p) >= v.TwoThirdsSignersCount()   Int comparison, don't use dec, it's less error-prone.

File: consensus/quorum/one-node-one-vote.go (hunk 1 above)
Comment (chaosma) [reply consensus/quorum/one-node-one-vote.go]: removed this function

File: consensus/quorum/one-node-one-vote.go (hunk 2 above)
Comment (chaosma) [reply consensus/quorum/one-node-one-vote.go]: yeah, use countsonebit

**Pull request merged**